easybuild/easyconfigs/m/MOABS/MOABS-1.3.2-foss-2019b.eb

```diff
@@ -0,0 +1,41 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'MakeCp'
+
+name = 'MOABS'
+version = '1.3.2'
+
+homepage = 'https://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-2-r38'
+description = """Model based analysis of bisulfite sequencing data"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+# https://code.google.com/archive/p/moabs/
+source_urls = ['https://s3.amazonaws.com/deqiangsun/software/%(namelower)s/']
+sources = ['%(namelower)s-v%(version)s.src.x86_64_Linux.data.no_database.tar.gz']
+checksums = ['9f4903182b803de381ae517847575d90faf3aed8d96b65999d619ce6b1bf8a4a']
+
+fix_perl_shebang_for = ['bin/*.pl']
+
+dependencies = [
+    ('Perl', '5.30.0'),
+    ('zlib', '1.2.11'),
+]
+
+local_bin_list = ['bamsort.sh', 'bsmap', 'mcall', 'mcomp', 'moabs', 'numCI', 'preprocess_novoalign.sh', 'redepth.pl']
+
+files_to_copy = ['bin']
+
+sanity_check_paths = {
+    'files': ['bin/%s' % f for f in local_bin_list],
+    'dirs': []
+}
+sanity_check_commands = [
+    ('bsmap -h 2>&1 | grep "bsmap \[options\]"', ''),
```

Comment:
No need for tuples with 2nd element an empty string (stuff to pass to stdin), just use strings:

```sanity_check_commands = [
    ("bsmap -h 2>&1 | grep 'bsmap \[options\]'",